### PR TITLE
[craftedv2beta] Module documentation for `crafted-startup`

### PR DESCRIPTION
**Pull request by sthesing**
Next module still lacking docs.

### CODE REVIEW

File: docs/crafted-startup.org [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+;; Load crafted-startup configuration
+(require 'crafted-startup-config)
+#+end_src
+
+** Description
+
+The =crafted-startup= module provides a fancy splash screen similar to the Emacs
+default splash screen or the Emacs about page.
+
+If it is used together with the =crafted-updates= module, it will also check for
```
Comment (jeffbowman) [inline docs/crafted-startup.org:23]: This may need an example source block similar to elucidate the issue.

**Review by jeffbowman — CHANGES_REQUESTED** [review]
Simple change requested.

File: docs/crafted-startup.org (hunk 1 above)
Comment (sthesing) [reply docs/crafted-startup.org:23]: Good call. Done.

**Review by jeffbowman — APPROVED** [review]

**Review by jvdydev — APPROVED** [review]

**Comment by jvdydev** [comment]
Looks good

**Pull request merged**